src/live/index.js

```diff
@@ -25,6 +26,21 @@ const onConnect = (io, socket) => {
   socket.on('chat:album', async (data) => {
     handleAlbumChat(io, socket, data);
   });
+
+  socket.on('listening-to-album-play', (albumId) => {
+    albumController.startAlbumPlayback(albumId);
+    console.log('Album is now playing');
+  });
+
+  socket.on('listening-to-album-pause', () => {
+    io.emit('album-paused', 'Album is now paused');
```

Comment:
Keep in mind that `emit` sends this message to _all_ sockets that have subscribed to the `album-paused` event. But right now the message we're emitting doesn't give those listeners much context about _which_ album was paused and by _whom_.

Reply:
Yes,  you  are right, it is true, should I delete it ?

Reply:
I think likely you'll end up combining the changes you have here with your other PR right? I think that one has a good state right now of setting up rooms and emmitting a more full set of info. As long as you end up merging those in as the final state it should be fine.